src/main/java/space/obminyashka/items_exchange/security/jwt/refresh/RefreshTokenService.java

```diff
@@ -0,0 +1,37 @@
+package space.obminyashka.items_exchange.security.jwt.refresh;
+
+import java.util.Optional;
+
+public interface RefreshTokenService {
```

Comment:
Prefer having that into a service layer according to MVC pattern.

Reply:
Got it